Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,17 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+const tasksInHours = tasks.map(function(x){
+  let y = x;
```

Comment:
Try to give meaningful names to your variables, it helps a lot with readability.